app/helpers/application_helper.rb

```diff
@@ -132,27 +132,17 @@ def no_hover_class(item)
   end
 
   # Check role based authorization for a UI task
-  def role_allows(options = {})
-    ApplicationHelper.role_allows_intern(options) rescue false
+  def role_allows(**options)
```

Comment:
@dclarizio circa 2007 might be able to tell you ;)
b0021863d7b9905599a937a495f044eb4e54a172

In any case, all the callers in the front end use this helper so changing it here wouldn't really be worth it IMO.